Unit testing for logging module added

kannankvs opened the pull request:
##### SUMMARY
Unit testing for logging module added

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
logging Unit Testing

kerry-meyer commented on tests/unit/modules/network/sonic/sonic_module.py:
Why does this function become necessary with the addition of the "logging" UT? (What does it do? Why wasn't it needed for UT in other resource modules?

kerry-meyer replied:
This file has been deleted from the change set.

I am marking this issue "Resolved".

kerry-meyer commented on tests/unit/modules/network/sonic/test_sonic_logging.py:
```diff
@@ -0,0 +1,87 @@
+        self.get_interface_naming_mode = self.mock_get_interface_naming_mode.start()
+        self.get_interface_naming_mode.return_value = 'standard'
+        self.utils_send_requests = self.mock_utils_send_requests.start()
+        self.utils_send_requests.side_effect = self.send_requests_side_effect
```
Related to the question asked on the sonic_module file: What is the purpose of this additional setup step?

kerry-meyer replied:
This code has been revised.

I am marking this issue "Resolved".

kerry-meyer reviewed (commented):
I have posted a couple of questions about the addition of the "send_requests_side_effect" function in the UT for this resource module. I am not expecting these to result in change requests, but would like to understand the functionality before approving and merging this PR.

kannankvs commented:
@kerry-meyer : Modified the mocking method for send_requests; reused config_side_effect instead of introducing new mocking method. Previously, I was thinking that mocking send_requests differently might be required in many modules. Its now removed and same old config_side_effect is reused.
